<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationChooser</class>
 <widget class="QWidget" name="ClassificationChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Classification Chooser</string>
  </property>
  <widget class="QComboBox" name="cboClassificationMethod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpKNN">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>KNN</string>
   </property>
   <widget class="QLineEdit" name="txtKnnK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblK">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpSVM">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>SVM</string>
   </property>
   <widget class="QComboBox" name="cboSvmKernel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblKernelType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDegree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSvmDegree">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSvmGamma">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>260</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
